addons/mail/static/src/model/tests/model_field_command_tests.js

```diff
@@ -0,0 +1,714 @@
+/** @odoo-module **/
+
+import {
+    clear,
+    create,
+    decrement,
+    increment,
+    insert,
+    insertAndReplace,
+    link,
+    replace,
+    unlink,
+    unlinkAll,
+} from 'mail/static/src/model/model_field_command.js';
+import {
+    afterEach,
+    beforeEach,
+    start,
+} from 'mail/static/src/utils/test_utils.js';
+
+QUnit.module('mail', {}, function () {
+QUnit.module('model', {}, function () {
+QUnit.module('model_field_command_tests.js', {
+    beforeEach(){
+        beforeEach(this);
+        this.start = async params => {
+            const { env, widget } = await start(Object.assign({}, params, {
+                data: this.data,
+            }));
+            this.env = env;
+            this.widget = widget;
+        };
+    },
+    afterEach() {
+        afterEach(this);
+    },
+});
+
+QUnit.test('clear: should set attribute field undefined', async function (assert) {
+    assert.expect(1);
+
+    await this.start();
+    this.env.models['test.task'].create({
+        id: 1,
+        title: 'test title 1',
+    });
+    const task = this.env.models['test.task'].findFromIdentifyingData({ id: 1 });
```

Comment:
You don't need to use `findFromIdentifyingData` when you create the record manually. Just assign value to the result of create.